apps/web/src/components/Shared/ChooseFile.tsx

```diff
@@ -11,6 +11,9 @@ const ChooseFile: FC<ChooseFileProps> = ({ onChange }) => {
       type="file"
       accept=".png, .jpg, .jpeg, .gif"
       onChange={onChange}
+      onClick={(event) => {
+        (event.target as HTMLInputElement).value = '';
```

Comment:
we can do something like this right? no?

`onClick={(event: HTMLInputElement) => {`

Reply:
I tried your suggestion but the function signature for `onClick` is `(event: MouseEvent<HTMLInputElement, MouseEvent>)`.  I couldn't get it working, but further suggestions appreciated, if you can find another way.

Regarding typecast, it seems that it is needed here. I think this is relevant: https://stackoverflow.com/a/57701995/961254

I'll add a comment though, because it's not obvious what this line does :)

Reply:
It's fine then, its not a big type issue 🙇🏼